pom.xml

```diff
@@ -122,7 +122,7 @@
     <!-- properties for script build step that creates the config files for the artifacts -->
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <app.parents>system:cdap-etl-batch[3.4.0,3.5.0-SNAPSHOT),system:cdap-etl-realtime[3.4.0,3.5.0-SNAPSHOT),system:cdap-data-pipeline[3.4.0,3.5.0-SNAPSHOT)</app.parents>
+    <app.parents>system:cdap-etl-batch[3.4.4-SNAPSHOT,3.5.0-SNAPSHOT),system:cdap-etl-realtime[3.4.4-SNAPSHOT,3.5.0-SNAPSHOT),system:cdap-data-pipeline[3.4.4-SNAPSHOT,3.5.0-SNAPSHOT)</app.parents>
```

Comment:
we shouldn't change the lower bound here. In case somebody upgrades from hydrator 3.4.0 to 3.4.4, we still want plugins from this branch to work with older 3.4.x versions.